internal/meta/label.go

```diff
@@ -0,0 +1,3 @@
+package meta
```

Comment:
This is empty now. But we'll define labels later when implementing Config CR reconciliation.